libs/auth/src/drivers/magento/transforms/auth-transformer.service.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { TestBed } from '@angular/core/testing';
+
+import { DaffMagentoAuthTransformerService } from './auth-transformer.service';
+import { DaffAuthTransformer } from '../../injection-tokens/auth-transformer.token';
+import { GenerateTokenResponse } from '../models/outputs/generate-token-response';
+
+describe('DaffMagentoAuthTransformerService', () => {
+  let service: DaffMagentoAuthTransformerService;
+  let mockResponse: GenerateTokenResponse;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
+        {
+          provide: DaffAuthTransformer,
+          useExisting: DaffMagentoAuthTransformerService
+        }
+      ]
+    })
+
+    service = TestBed.get(DaffAuthTransformer);
+    mockResponse = {
+      generateCustomerToken: {
+        token: 'sdaf87sadbnfnsd'
+      }
+    };
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  describe('transform | transforming the token response into a DaffAuthToken', () => {
```

Comment:
I think you should be an object comparison if possible:
```
let mockToken = 'sdaf87sadbnfnsd';
mockResponse = {
   generateCustomerToken: {
     token: mockToken
   }
};
expectedDaffAuthToken {
  token: mockToken
};

describe('transform', () => {
  it('should transform a magento token response into a DaffAuthToken', () => {
    const result: DaffAuthToken = service.transform(mockResponse);

    expect(result).toEqual(expectedDaffAuthToken);
  });
});
```

Reply:
changed to this.